src/main/java/de/tum/in/www1/artemis/repository/ExamUserRepository.java

```diff
@@ -20,5 +21,22 @@ public interface ExamUserRepository extends JpaRepository<ExamUser, Long> {
             """)
     ExamUser findByExamIdAndUserId(@Param("examId") long examId, @Param("userId") long userId);
 
-    List<ExamUser> findAllByExamId(long examId);
+    List<ExamUser> findAllByExamId(@Param("examId") long examId);
+
+    @Query("""
+            SELECT new de.tum.in.www1.artemis.web.rest.dto.ExamUserAttendanceCheckDTO(examUser.id, examUser.studentImagePath, examUser.user.login,
```

Comment:
Should we do the object creation directly here or move this into a service?

Reply:
I think we need to do it here as we need to mix information from `exam_user` and `students_exam` tables. Do you know how to create this object from a service?